images/win/scripts/Installers/Install-DebianImage.ps1

```diff
@@ -0,0 +1,29 @@
+################################################################################
+##  File:  Install-DebianImage.ps1
+##  Desc:  Install Debian Image for WSL
+################################################################################
+
+Import-Module -Name ImageHelpers
+
+debianTempDir = "C:\Temp\debian"
+
+if (-not (Test-Path $debianPath)) {
+    New-Item -Path debianTempDir -ItemType Directory
+}
+
+Push-Location debianTempDir
+
+$debianZipName = "debian.zip"
+Invoke-WebRequest -Uri "https://aka.ms/wsl-debian-gnulinux" -OutFile $debianZipName
```

Comment:
curious why did you choose Debian?

Reply:
The only reason to use Debian is because this image takes up less space than others.
I don't see any objections to using Ubuntu instead, so if that makes sense, let's use it